federation/ldap/src/main/java/org/keycloak/storage/ldap/LDAPStorageProvider.java

```diff
@@ -343,53 +346,59 @@ public UserModel getUserById(RealmModel realm, String id) {
 
     @Override
     public int getUsersCount(RealmModel realm) {
-        return 0;
+        return getUsersCount(realm, Collections.emptyMap());
```

Comment:
I think we should create a follow-up issue for all count queries for LDAP. Currently, it is really inefficient. I think we need to find some solution to that. Maybe we can just throw UnsupportedException all the time and adjust code calling that method to recover from that.

Reply:
https://github.com/keycloak/keycloak/issues/20156